### ISSUE TITLE
使用buildtool时发生错误

### ISSUE THREAD

**Issue by BlueHeart6** [ApolloAuto/apollo#15287]
运行buildtool build -p core build application-pnc时发生错误 请问是什么原因呢
![_cgi-bin_mmwebwx-bin_webwxgetmsgimg  MsgID=7560635494051884652 skey=@crypt_e230addb_de52ecb4d4fa28aff7267f6aa6d9b45e mmweb_appid=wx_webfilehelper](https://github.com/ApolloAuto/apollo/assets/142152627/68051956-e9c4-49bd-a1cf-b5c9eea96092)

**Comment by daohu527** [comment]
The reason for this problem is that downloading the dependencies of bazel failed. This is mostly due to network reasons

## How to slove
Here is a document on how to solve it. [link](https://github.com/ApolloAuto/apollo/blob/3859bee50e92d81e94f64ec4a3b3d4241978e8f5/docs/01_Installation%20Instructions/how_to_prepare_bazel_dist_dir.md)

To put it simply, you can manually download this file, put it in `/apollo/.cache/distdir`, and then recompile.

**Comment by BlueHeart6** [comment]
我的bazel版本为5.3.2 似乎还无法通过wget 'https://apollo-system.cdn.bcebos.com/archive/bazel_deps/bazel-dependencies-5.3.2.tar.gz'下载依赖 可能是apollo9.0还没有进行相关更新

**Comment by daohu527** [comment]
Yes, so you can only download manually if you encounter a download error. Can you access the external network?

If no, we will update the package soon

**Comment by WildBeast114514** [comment]
可以试下这个缓存， https://apollo-system.cdn.bcebos.com/bazel_deps/5.2.1/build.tar.gz
解压后替换工作空间的.cache文件
